web/src/settings_components.ts

```diff
@@ -424,6 +441,22 @@ export function read_field_data_from_form(
     return undefined;
 }
 
+function get_field_data_input_value($input_elem: JQuery): string | undefined {
+    const $profile_field_form = $input_elem.parent();
```

Comment:
Can we use `.closest("profile-field-form")` here instead of `parent()`? That will help in avoiding bugs if we change the HTML structure for some reason in future.

Reply:
Fixed [here](https://github.com/zulip/zulip/compare/fb714ff6ffa7d12ade6dc359b4fdf7e77980fa77..8b36a3f8d4a74997d8760925d67f7f7fc646c0f9)